src/content/guides/code-splitting.md

```diff
@@ -97,79 +92,78 @@ module.exports = {
 This will yield the following build result:
 
 ``` bash
-Hash: 309402710a14167f42a8
-Version: webpack 2.6.1
-Time: 570ms
-            Asset    Size  Chunks                    Chunk Names
-  index.bundle.js  544 kB       0  [emitted]  [big]  index
-another.bundle.js  544 kB       1  [emitted]  [big]  another
-   [0] ./~/lodash/lodash.js 540 kB {0} {1} [built]
-   [1] (webpack)/buildin/global.js 509 bytes {0} {1} [built]
-   [2] (webpack)/buildin/module.js 517 bytes {0} {1} [built]
-   [3] ./src/another-module.js 87 bytes {1} [built]
-   [4] ./src/index.js 216 bytes {0} [built]
+Hash: a948f6cc8219cc2d39a1
+Version: webpack 4.7.0
```

Comment:
This is not related with this change, but I have been thinking about including these logs in our documentation, some examples have a different version than others, I am not sure if that is good or bad, but I find it confusing.

Maybe we should remove them?

Sorry for the noise :)

Reply:
i think it can confuse the readers at least if previous page said `Version webpack 3.11.0` and next page says `... 4.7.0` so maybe its worth to consider cleaning output logs from that through the docs. Lets raise this one in slack thread maybe?

Reply:
Yeah

Reply:
I agree, we should eliminate all this logs since maintaining them is tons of work. I'll create an issue so we can tackle it later.